<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>633</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy>
    <hsizetype>5</hsizetype>
    <vsizetype>5</vsizetype>
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle" >
   <string>MessageMonitor</string>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <layout class="QVBoxLayout" >
    <property name="margin" >
     <number>9</number>
    </property>
    <property name="spacing" >
     <number>6</number>
    </property>
    <item>
     <widget class="QTabWidget" name="tabWidget" >
      <property name="enabled" >
       <bool>true</bool>
      </property>
      <property name="tabPosition" >
       <enum>QTabWidget::South</enum>
      </property>
      <property name="currentIndex" >
       <number>0</number>
      </property>
      <widget class="QWidget" name="tabMessages" >
       <attribute name="title" >
        <string>Messages</string>
       </attribute>
       <layout class="QVBoxLayout" >
        <property name="margin" >
         <number>9</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item>
         <widget class="QSplitter" name="splitterMessage" >
          <property name="orientation" >
           <enum>Qt::Vertical</enum>
          </property>
          <widget class="QTreeWidget" name="treeMessages" >
           <property name="autoScroll" >
            <bool>false</bool>
           </property>
           <property name="alternatingRowColors" >
            <bool>true</bool>
           </property>
           <property name="selectionMode" >
            <enum>QAbstractItemView::SingleSelection</enum>
           </property>
           <property name="rootIsDecorated" >
            <bool>false</bool>
           </property>
           <property name="uniformRowHeights" >
            <bool>true</bool>
           </property>
          </widget>
          <widget class="QTreeWidget" name="treeClients" >
           <property name="alternatingRowColors" >
            <bool>true</bool>
           </property>
           <property name="selectionMode" >
            <enum>QAbstractItemView::SingleSelection</enum>
           </property>
           <property name="rootIsDecorated" >
            <bool>false</bool>
           </property>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabClients" >
       <attribute name="title" >
        <string>Clients</string>
       </attribute>
       <layout class="QVBoxLayout" >
        <property name="margin" >
         <number>9</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item>
         <widget class="QTreeWidget" name="treeActiveClients" >
          <property name="alternatingRowColors" >
           <bool>true</bool>
          </property>
          <property name="sortingEnabled" >
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabStatistics" >
       <attribute name="title" >
        <string>Statistics</string>
       </attribute>
       <layout class="QVBoxLayout" >
        <property name="margin" >
         <number>9</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item>
         <widget class="QSplitter" name="splitterStatistics" >
          <property name="orientation" >
           <enum>Qt::Vertical</enum>
          </property>
          <widget class="QTreeWidget" name="treeStats" >
           <property name="selectionMode" >
            <enum>QAbstractItemView::NoSelection</enum>
           </property>
           <column>
            <property name="text" >
             <string>Name</string>
            </property>
           </column>
           <column>
            <property name="text" >
             <string>Messages</string>
            </property>
           </column>
           <column>
            <property name="text" >
             <string>Data</string>
            </property>
           </column>
           <item>
            <property name="text" >
             <string>Clients</string>
            </property>
            <property name="text" >
             <string/>
            </property>
            <property name="text" >
             <string/>
            </property>
           </item>
           <item>
            <property name="text" >
             <string>Groups</string>
            </property>
            <property name="text" >
             <string/>
            </property>
            <property name="text" >
             <string/>
            </property>
           </item>
           <item>
            <property name="text" >
             <string>Overall</string>
            </property>
            <property name="text" >
             <string/>
            </property>
            <property name="text" >
             <string/>
            </property>
            <item>
             <property name="text" >
              <string/>
             </property>
             <property name="text" >
              <string>0</string>
             </property>
             <property name="text" >
              <string>0</string>
             </property>
            </item>
           </item>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabSettings" >
       <attribute name="title" >
        <string>Settings</string>
       </attribute>
       <layout class="QVBoxLayout" >
        <property name="margin" >
         <number>9</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item>
         <widget class="QSplitter" name="splitter" >
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_3" >
          <property name="sizePolicy" >
           <sizepolicy>
            <hsizetype>7</hsizetype>
            <vsizetype>4</vsizetype>
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="title" >
           <string>GUI</string>
          </property>
          <layout class="QGridLayout" >
           <property name="margin" >
            <number>9</number>
           </property>
           <property name="spacing" >
            <number>6</number>
           </property>
           <item row="1" column="0" colspan="2" >
            <layout class="QHBoxLayout" >
             <property name="margin" >
              <number>0</number>
             </property>
             <property name="spacing" >
              <number>6</number>
             </property>
             <item>
              <spacer>
               <property name="orientation" >
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" >
                <size>
                 <width>310</width>
                 <height>27</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QPushButton" name="buttonApply" >
               <property name="text" >
                <string>Apply</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="0" column="0" >
            <widget class="QLabel" name="label_7" >
             <property name="text" >
              <string>Message count:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1" >
            <widget class="QLineEdit" name="editMessageCount" >
             <property name="text" >
              <string>100</string>
             </property>
             <property name="alignment" >
              <set>Qt::AlignRight</set>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_2" >
          <property name="sizePolicy" >
           <sizepolicy>
            <hsizetype>5</hsizetype>
            <vsizetype>5</vsizetype>
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="title" >
           <string>Log</string>
          </property>
          <layout class="QGridLayout" >
           <property name="margin" >
            <number>9</number>
           </property>
           <property name="spacing" >
            <number>6</number>
           </property>
           <item row="0" column="0" >
            <widget class="QTextEdit" name="textLog" >
             <property name="sizePolicy" >
              <sizepolicy>
               <hsizetype>5</hsizetype>
               <vsizetype>3</vsizetype>
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="undoRedoEnabled" >
              <bool>false</bool>
             </property>
             <property name="lineWrapMode" >
              <enum>QTextEdit::NoWrap</enum>
             </property>
             <property name="readOnly" >
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>611</y>
     <width>500</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="toolBar" >
   <property name="orientation" >
    <enum>Qt::Horizontal</enum>
   </property>
   <attribute name="toolBarArea" >
    <number>4</number>
   </attribute>
   <addaction name="actionConsoleLog" />
   <addaction name="actionAutoOpenMessage" />
   <addaction name="actionClearMessages" />
  </widget>
  <widget class="QMenuBar" name="menuBar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>500</width>
     <height>29</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile" >
    <property name="title" >
     <string>&amp;File</string>
    </property>
    <addaction name="separator" />
    <addaction name="actionQuit" />
   </widget>
   <widget class="QMenu" name="menuPlugins" >
    <property name="enabled" >
     <bool>false</bool>
    </property>
    <property name="title" >
     <string>&amp;Plugins</string>
    </property>
   </widget>
   <addaction name="menuFile" />
   <addaction name="menuPlugins" />
  </widget>
  <action name="actionConsoleLog" >
   <property name="checkable" >
    <bool>true</bool>
   </property>
   <property name="text" >
    <string>Enable console &amp;log</string>
   </property>
  </action>
  <action name="actionAutoOpenMessage" >
   <property name="checkable" >
    <bool>true</bool>
   </property>
   <property name="text" >
    <string>Auto open incoming message</string>
   </property>
  </action>
  <action name="actionQuit" >
   <property name="text" >
    <string>&amp;Quit</string>
   </property>
  </action>
  <action name="actionClearMessages" >
   <property name="text" >
    <string>Clear messages</string>
   </property>
   <property name="iconText" >
    <string>Clear messages</string>
   </property>
   <property name="toolTip" >
    <string>Clear messages</string>
   </property>
  </action>
 </widget>
 <pixmapfunction></pixmapfunction>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>treeMessages</tabstop>
  <tabstop>treeClients</tabstop>
  <tabstop>treeActiveClients</tabstop>
  <tabstop>treeStats</tabstop>
  <tabstop>editMessageCount</tabstop>
  <tabstop>buttonApply</tabstop>
  <tabstop>textLog</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
